hiera.tf

```diff
@@ -0,0 +1,14 @@
+module "hiera" {
+  source = "./modules/hiera"
```

Comment:
Probably makes sense to move this one to github.com/appuio/terarform-modules.git as well

Reply:
Merged the modules back together since we use the same template.